release_notes/v1.2.md

```diff
@@ -0,0 +1,349 @@
+    - <b>Enabled user-customized tool subsets:</b> Added support for [user-defined subsets of tools](https://github.com/opea-project/GenAIComps/issues/978) for the ChatCompletion API and Assistant APIs.
+    - <b>Enabled persistence:</b> Introduced Redis to persist Agent configurations and historical messages for Agent recovery and multi-turn conversations.
+- <b>Long-context Summarization:</b> Supported multiple modes: [`auto`](https://github.com/opea-project/GenAIComps/commit/5aba3b25), [`stuff`, `truncate`, `map_reduce`, and `refine`](https://github.com/opea-project/GenAIComps/commit/45d00020).
+- <b>Standalone Microservice Deployment</b>: Enabled the deployment of OPEA components as independent services, allowing for greater flexibility, scalability, and modularity in various application scenarios.
+- <b>[PDF Inputs Support](https://github.com/opea-project/GenAIComps/pull/1134):</b> Support PDF inputs for dataprep, embeddings, LVMs, and retrievers.
+
+#### New GenAI Components
+- <b>[Bedrock](https://github.com/opea-project/GenAIComps/pull/1031):</b> OPEA LLM now supports [Amazon Bedrock](https://aws.amazon.com/bedrock/) as the backend of the text generation microservice. Bedrock is a fully managed service that offers a choice of high-performing foundation models (FMs) from leading AI companies through a single API, along with a broad set of capabilities you need to build generative AI applications with security, privacy, and responsible AI.
+- <b>[OpenSearch Vector Database](https://github.com/opea-project/GenAIComps/pull/1024):</b> OPEA vectorstores now supports [AWS OpenSearch](https://aws.amazon.com/opensearch-service/). OpenSearch is an open-source, enterprise-grade search and observability suite that brings order to unstructured data at scale.
+- <b>[Elasticsearch Vector Database](https://github.com/opea-project/GenAIComps/pull/972):</b> OPEA vectorestores now supports [Elasticsearch vector database](https://www.elastic.co/elasticsearch/vector-database), Elasticsearch's open source vector database offering an efficient way to create, store, and search vector embeddings.
+- <b>[Guardrail Hallucination Detection](https://github.com/opea-project/GenAIComps/pull/1108):</b> Added the capability of detecting Hallucination which spans a wide range of issues that can impact reliability, trustworthiness, and utility of AI-generated content. 
+
+#### Enhanced GenAI Examples
+- <b>ChatQnA</b>: Enabled [embedding and reranking on vLLM](https://github.com/opea-project/GenAIExamples/issues/1203), and [Jaeger UI and OpenTelemetry tracing](https://github.com/opea-project/GenAIExamples/pull/1316) for TGI serving on HPU.
```

Comment:
Please check the linked items.  https://github.com/opea-project/GenAIExamples/issues/1203 is still open, and tagged for v1.3 release, not v1.2.